styles/alphav3-1.json

```diff
@@ -167,24 +180,38 @@
     "custom": {}
   },
   "styles": {
+    "spacing": {
+      "padding": {
+        "top": "0px",
+        "right": "inherit",
+        "bottom": "0px",
+        "left": "inherits"
```

Comment:
"inherits" or "inherit"?

Reply:
haha, obviously it doesn't seem like it was doing anything.